### PR TITLE
update ngChosen to be able to take ngDisabled attribute

### PR DESCRIPTION
**Pull request by a-morales**
@sebasrodriguez @jakhil @matiasdelgado 

Updating ngChosen to be able to be disabled if the `ngDisabled` attribute is passed to it.

### CODE REVIEW

**Comment by a-morales** [comment]
@sebasrodriguez @matiasdelgado @jakhil 

Updated ngChosen to be able to be updated if the model is cleared from an outside event.

File: src/ngChosen.ts [hunk 1]
```diff
@@ -74,16 +75,25 @@ namespace ngChosen {
...
             } else {
-                this.updateState(elem, false, false, false);
+                this.updateState(elem, false, (!angular.isUndefined(scope.ngDisabled) && scope.ngDisable), false);
+            }
+
+            if (scope.ngDisabled !== undefined) {
+                scope.$watch("ngDisabled",(newValue, oldValue) => {
+                    if (!angular.isUndefined(newValue) && newValue !== oldValue) {
+                        this.updateState(elem, false, newValue, false);
+                        this.triggerUpdate(elem);
```
Comment (jakhil) [inline src/ngChosen.ts]: Is this step necessary, as 'updateState' internally calls the 'triggerUpdate' function.

Comment (a-morales) [inline src/ngChosen.ts]: Good catch, just fixed it.

**Comment by jakhil** [comment]
Looks good to me.
:+1:

**Comment by sebasrodriguez** [comment]
Good job guys. Merging the PR.

**Pull request merged**